tasks/brand.js

```diff
@@ -0,0 +1,12 @@
+module.exports = (gulp, config) => () => {
+
+  const git = require('git-promise');
+  return git('remote add assets ' + config.cmdParams.repo).then(function () {
+    return git('pull assets ' + config.cmdParams.branch + ' --squash --allow-unrelated-histories').then(function(){
+      return git('add ./assets').then(function(){
+        return git('commit --m \'Updated assets with branding \'' + config.cmdParams.branch);
```

Comment:
Will this not add the commit to the "infinispan-management-console" repo? Or is on the remote repo?